kitsune/questions/jinja2/questions/includes/question_editing_frame.html

```diff
@@ -90,12 +90,20 @@ <h1 class="sumo-page-heading">{% block headline %}{% endblock %}</h1>
               {% endif %}
 
               <div class="info card shade-bg highlight mb">
+              {% if current_product.key == 'mozilla-account' %}
+                {% trans %}
+                  Can't sign in to your account and need help? 
+                  You've found the right place.  Complete the form below
```

Comment:
Let's remove the extra space

Reply:
I am unclear what space you mean.

Reply:
`You've found the right place.  Complete the form below`

Between the dot and `Complete` there are 2 spaces instead of one

Reply:
Got it!